crates/rpc/src/handlers/version.rs

```diff
@@ -0,0 +1,18 @@
+use serde_json::json;
+use warp::{
+    http::status::StatusCode,
+    reject::Rejection,
+    reply::{Reply, json, with_status},
+};
+
+const APPLICATION_NAME: &str = "Ream";
+
+/// Called by `/version` to get the Node Version.
+pub async fn get_version() -> Result<impl Reply, Rejection> {
+    let version = format!("{}{}{}", APPLICATION_NAME, "/v", env!("CARGO_PKG_VERSION"));
+
+    Ok(with_status(
+        json(&json!({"data":{"version":version}})),
+        StatusCode::OK,
+    ))
+}
```

Comment:
Hmm I don't like this way of getting the version string, I will make a PR tomorrow for generating it, then you can rebase your PR and use my code :+1:

Reply:
I will just make a PR for it, as it is a little involved